changelogs/unreleased/6475-prevent-too-eager-overwrite.yml

```diff
@@ -0,0 +1,4 @@
+description: Fix for state overwriting introduced by #6486
```

Comment:
I think the ticket reference might be a comment in yaml, not sure.

Reply:
ooh, good catch